dash/App/Chains/Chain/Components/index.js

```diff
@@ -0,0 +1,213 @@
+      <div className='chainColorSwatches'>
+        {[ 'accent1', 'accent2', 'accent3', 'accent4', 'accent5', 'accent6', 'accent7', 'accent8'].map(color => {
+          return (
+            <div
+              key={color}
+              className={currentColor === color ? 'chainColorSwatch chainColorSwatchSelected' : 'chainColorSwatch'}
+              style={{ background: `var(--${color})` }}
+              onClick={() => onChange(color)}
+            />
+          )
+        })}
+      </div>
+    </div>
+  )
+}
+
+export const EditChainName = ({currentName, onChange}) => {
+  return (
+    <div className='chainRow'>
+      <label htmlFor='chainName' className='chainInputLabel'>Chain Name</label>
+      <input
+        id='chainName'
+        className={currentName === networkDefaults.name ? 'chainInput chainInputDim' : 'chainInput'}
+        value={currentName}
+        spellCheck='false'
+        onChange={(e) => onChange(e.target.value)}
+        onFocus={(e) => {
+          if (e.target.value === networkDefaults.name) onChange('')
+        }}
+        onBlur={(e) => {
+          if (e.target.value === '') onChange(networkDefaults.name)
+        }}
+      />
+    </div>
+  )
+}
+
+export const EditChainSymbol = ({ currentSymbol, onChange }) => {
+  return (
+    <div className='chainRow'>
+      <label htmlFor='chainSymbol' className='chainInputLabel'>Native Symbol</label>
+      <input
+        id='chainSymbol'
+        className={currentSymbol === networkDefaults.symbol ? 'chainInput chainInputDim' : 'chainInput'}
+        value={currentSymbol}
+        spellCheck='false'
+        onChange={(e) => {
+          if (e.target.value.length > 8) return e.preventDefault()
+          onChange(e.target.value)
+        }}
+        onFocus={(e) => {
+          if (e.target.value === networkDefaults.symbol) onChange('')
+        }}
+        onBlur={(e) => {
+          if (e.target.value === '') onChange(networkDefaults.symbol)
+        }}
+      />
+    </div>
+  )
+}
+
+export const EditChainId = ({ chainId, onChange }) => {
+  return (
+    <div className='chainRow'>
+      <label htmlFor='chainId' className='chainInputLabel'>Chain ID</label>
+      <input
+        id='chainId'
+        className={chainId === networkDefaults.id ? 'chainInput chainInputDim' : 'chainInput'}
+        value={chainId}
+        spellCheck='false'
+        onChange={(e) => {
+          if (Number(parseInt(e.target.value)) || e.target.value === '') {
+            onChange(e.target.value)
+          }
+        }}
+        onFocus={(e) => {
+          if (e.target.value === networkDefaults.id) onChange('')
+        }}
+        onBlur={(e) => {
+          if (e.target.value === '') onChange(networkDefaults.id)
+        }}
+      />
+    </div>
+  )
+}
+
+export const EditTestnet = ({ testnet, onChange }) => {
+  return (
+    <div className='chainRow chainRowTestnet'>
+      <label id='testnet-label' className=''>Is this chain a testnet?</label>
```

Comment:
can't put my finger on it but don't love this verbiage. could we stick w. something simpler like "Testnet?" similar to what chainlist.org has?